src/AltinnCore/Runtime/Controllers/PartiesController.cs

```diff
@@ -65,10 +65,18 @@ public class PartiesController : ControllerBase
         /// </summary>
         /// <returns></returns>
         [HttpGet("{org}/{app}/api/v1/parties")]
-        public IActionResult Get()
+        public IActionResult Get(string org, string app, bool allowedToInstantiate = false)
```

Comment:
Could we rename parameter to: allowedToInstantiateFilter? That way we say true or false to it being enabled.  
Currently, it may come across as only return instances not allowed to instantiate if default is used.